sources/install/package_ad.sh

```diff
@@ -125,6 +125,53 @@ function install_bloodhound() {
     add-to-list "bloodhound,https://github.com/BloodHoundAD/BloodHound,Active Directory security tool for reconnaissance and attacking AD environments."
 }
 
+function install_bloodhound-ce() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing BloodHound-CE"
+
+    colorecho "Installing and configuring the database"
+    fapt postgresql postgresql-client
+    service postgresql start
+    cd /tmp
+    sudo -u postgres psql -c "CREATE USER bloodhound WITH PASSWORD 'bloodhoundcommunityedition';"
+    sudo -u postgres psql -c "CREATE DATABASE bloodhound;"
+    sudo -u postgres psql -c "GRANT ALL PRIVILEGES ON DATABASE bloodhound TO bloodhound;"
+    service postgresql restart
+
+    colorecho "Installing and configuring the Bloodhound-CE"
+    git -C /opt/tools clone --depth 1 https://github.com/SpecterOps/BloodHound.git
+    cd /opt/tools/BloodHound
+    python3 -m venv ./venv
+    cd ./packages/javascript/bh-shared-ui
+    yarn install
+    yarn build
+    cd /opt/tools/BloodHound
+    ./venv/bin/python3 ./packages/python/beagle/main.py build bh-ui -v
+    ./venv/bin/python3 ./packages/python/beagle/main.py build bh -v -d
+    wget https://github.com/BloodHoundAD/SharpHound/releases/download/v2.0.0/SharpHound-v2.0.0.zip -O sharphound-v2.0.0.zip
+    sha256sum sharphound-v2.0.0.zip > sharphound-v2.0.0.zip.sha256
+    mkdir ./azurehound
+    cd ./azurehound
+    wget \
+    https://github.com/BloodHoundAD/AzureHound/releases/download/v2.0.5/azurehound-linux-amd64.zip \
+    https://github.com/BloodHoundAD/AzureHound/releases/download/v2.0.5/azurehound-linux-arm64.zip
+    cd /opt/tools/BloodHound
+    cp ./dist/bhapi ./bloodhound
+    mkdir -p /etc/bloodhound/collectors/sharphound/
+    cp -r ./sharphound-v2.0.0.zip* /etc/bloodhound/collectors/sharphound/
+    mkdir -p /etc/bloodhound/collectors/azurehound/
+    cp -r ./azurehound/azurehound-linux-a* /etc/bloodhound/collectors/azurehound/
```

Comment:
In my opinion, this should be handled by Exegol-resources

Reply:
Right ! We can install sharphound + azurehound from my_resources and link to /etc/bloodhound/collectors after

Reply:
Not 100% necessary to link it there is it?
But yeah let’s have the install made by exegol-resources

Reply:
We have to link because bloodhound needs these zip to work (when starts). Note : We can change the Path if necessary